Fixed post images with WP 6.7

girishpanchal30 opened the pull request:
Closes https://github.com/Codeinwp/otter-blocks/issues/2457

### Summary
In this PR, I've fixed the post images issue with the latest version of WordPress.

### Checklist before the final review

- [ ] Included E2E or unit tests for the changes in this PR.
- [x] Visual elements are not affected by independent changes.
- [x] It is at least compatible with the [minimum WordPress version](https://wordpress.org/plugins/otter-blocks/).
- [x] It loads additional script in frontend only if it is required.
- [x] Does not impact the [Core Web Vitals](https://web.dev/vitals/).
- [x] In case of deprecation, old blocks are safely migrated.
- [x] It is usable in Widgets and FSE.
- [x] Copy/Paste is working if the attributes are modified.
- [x] PR is following [the best practices]()

Soare-Robert-Daniel reviewed (changes requested):
One problem with eliminating `--image-ratio` is that it will make this option broken

<img width="1425" alt="Screenshot 2024-11-28 at 10 28 59" src="https://github.com/user-attachments/assets/a9259b04-4eb1-4c78-81cd-7bcd9f58e01b">

girishpanchal30 commented:
@Soare-Robert-Daniel I’ve reverted the `--image-ratio` CSS and resolved the issue through an alternative approach.

### Solution

WordPress 6.7 introduced the new feature, [Auto Sizes for Lazy Loaded Images](https://make.wordpress.org/core/2024/10/18/auto-sizes-for-lazy-loaded-images-in-wordpress-6-7/), which caused an issue with the post grid block images. To address this, I’ve removed the auto sizes by using a filter for the post grid block.

Thanks!

Soare-Robert-Daniel commented on src/blocks/blocks/posts/style.scss:
```diff
@@ -64,6 +64,7 @@
 	--pag-cont-margin: 10px 0 0 0;
 
 	border: none;
+	max-width: inherit !important;
```
One more clarification: Is this rule with `!important` still valid after you reverted to the previous solution? If only that filter were the problem, we could remove this change.

girishpanchal30 replied:
@Soare-Robert-Daniel I've removed the `max-width` and it seems working fine.
